cli_tools/common/utils/paramhelper/region.go

```diff
@@ -0,0 +1,18 @@
+package paramhelper
```

Comment:
What about calling this package "parameters"? utils\parameters are utils related to parameters.

Reply:
We have already had  "param" under "utils". This one is used by "param" and "storageutils".